themes/timer-hugo/layouts/allpublications/list.html

```diff
@@ -0,0 +1,24 @@
+{{ define "main" }}
+
+{{ partial "page-title.html" . }}
+
+{{ if .Site.Data.list_publications.enable }}
+{{ "<!-- Complete List Publications Section Start -->" | safeHTML }}
+<section id="publications">
+    <div class="container">
+        <div class="section-heading" style="margin-top:30px;margin-bottom:20px">
+            {{ with .Site.Data.list_publications.title }}<h2 style="text-align:center">{{ . }}</h2>{{ end }}
+        </div>
+        <ol style="text-align:justify">
+            {{ range $index, $element:= .Site.Data.list_publications.paper }}
```

Comment:
It might be interesting to add more fields such as the publication date (`YYYY-MM-DD`, so you can sort the list automatically at rendering time instead of trusting the YML database order. Totally optional and up to your taste!